### PR TITLE
chore(license): Update licensing year

### PR DESCRIPTION
**Pull request by codershiba**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) 2017-2020 Denis Fäcke
+Copyright (c) 2017-2024 Denis Fäcke
```
Comment (mcollina) [inline LICENSE:3]: This is incorrect. We should add a new line with "The Fastify Contributors" instead, starting from the date of fork.

**Review by mcollina — CHANGES_REQUESTED** [review]

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,9 @@
 The MIT License
 
-Copyright (c) 2017-2020 Denis Fäcke
+Copyright (c) 2017-2024 The Fastify Team
```
Comment (gurgunday) [inline LICENSE:3]:
```suggestion
Copyright (c) 2021-2024 The Fastify Team
```

Comment (gurgunday) [reply LICENSE:3]: This is the year of the fork

Comment (Uzlopak) [reply LICENSE:3]: It is not even what matteo requested.
He wrote to add another line (below of Denis Fäckes Line)  

```suggestion
Copyright (c) 2017-2020 Denis Fäcke
Copyright (c) 2021-2024 The Fastify Team
```

**Review by gurgunday — APPROVED** [review]
lgtm

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**